src/server/models/types.ts

```diff
@@ -70,6 +70,19 @@ export interface List extends PrismaClient.List {
   lastAnnualReviewStartDate: Date | null;
 }
 
+export interface ListsForDashboard extends PrismaClient.ListsForDashboard {
```

Comment:
do we actually change the type in any way? does the generated type not produce something like this?

Reply:
Good point, this is not necessary. I had issue with the type before so I created this as a temporary type but we shouldn't need it anymore